plutus-core/untyped-plutus-core/test/Evaluation/Builtins/Definition.hs

```diff
@@ -218,8 +225,10 @@ test_FailingSucc =
                 apply () (builtin () $ Right FailingSucc) $
                     mkConstant @Integer @DefaultUni @DefaultFunExt () 0
         typeErrOrEvalExcOrRes :: Either _ (Either BuiltinErrorCall _) <-
-            -- Here we rely on 'typecheckAnd' lazily running the action after type checking the term.
-            traverse (try . evaluate) $ typecheckEvaluateCek def defaultBuiltinCostModelExt term
```

Comment:
Ironically enough, we were also relying on `typecheckAnd` running the action strictly. Yes, both lazily and strictly, just in different ways, and I got a failure when I made `typecheckEvaluateCek` lazier. I've changed this test so that it's only sensitive to laziness and not strictness.